packages/terra-clinical-header/src/terra-dev-site/doc/example/HeaderLongTextWithContent.scss

```diff
@@ -0,0 +1,5 @@
+:local {
```

Comment:
even this one can be reused.

Reply:
Lets create a common sass file to share across the examples and lets drop the @extends syntax